spyder/dependencies.py

```diff
@@ -8,10 +8,128 @@
 
 
 import os
+import sys
 
 # Local imports
 from spyder.utils import programs
-
+from spyder.config.base import _
+
+
+CLOUDPICKLE_REQVER = '>=0.5.0'
+PYGMENTS_REQVER = '>=2.0'
+QTCONSOLE_REQVER = '>=4.5.5'
+NBCONVERT_REQVER = '>=4.0'
+SPHINX_REQVER = '>=0.6.6'
+PYLINT_REQVER = '>=0.25'
+PSUTIL_REQVER = '>=0.3'
+QTAWESOME_REQVER = '>=0.5.7'
+QTPY_REQVER = '>=1.5.0'
+PICKLESHARE_REQVER = '>=0.4'
+PYZMQ_REQVER = '>=17'
+CHARDET_REQVER = '>=2.0.0'
+NUMPYDOC_REQVER = '>=0.6.0'
+SPYDER_KERNELS_REQVER = '>=1.4.0'
+QDARKSTYLE_REQVER = '>=2.7'
+ATOMICWRITES_REQVER = '>=1.2.0'
+DIFF_MATCH_PATCH_REQVER = '>=20181111'
+WATCHDOG_REQVER = None
+KEYRING_REQVER = None
+PEXPECT_REQVER = '>=4.4.0'
+PARAMIKO_REQVER = '>=2.4.0'
+PYXDG_REQVER = '>=0.26'
+PYMPLER_REQVER = None
+
+DEPENDENCIES_BASE = [
+    {'modname': "cloudpickle",
+     'package_name': "cloudpickle",
+     'features': _("Serialize variables in the IPython kernel to send to Spyder."),
+     'required_version': CLOUDPICKLE_REQVER},
+    {'modname': "pygments",
+     'package_name': "pygments",
+     'features': _("Syntax highlighting for Matlab, Julia and other file types"),
+     'required_version': PYGMENTS_REQVER},
+    {'modname': "qtconsole",
+     'package_name': "qtconsole",
+     'features': _("Integrate the IPython console"),
+     'required_version': QTCONSOLE_REQVER},
+    {'modname': "nbconvert",
+     'package_name': "nbconvert",
+     'features': _("Manipulate Jupyter notebooks on the Editor"),
+     'required_version': NBCONVERT_REQVER},
+    {'modname': "sphinx",
+     'package_name': "sphinx",
+     'features': _("Show help for objects in the Editor and Consoles in a dedicated pane"),
+     'required_version': SPHINX_REQVER},
+    {'modname': "pylint",
+     'package_name': "pylint",
+     'features': _("Static code analysis"),
+     'required_version': PYLINT_REQVER},
+    {'modname': "psutil",
+     'package_name': "psutil",
+     'features': _("CPU and memory usage info in the status bar"),
+     'required_version': PSUTIL_REQVER},
+    {'modname': "qtawesome",
+     'package_name': "qtawesome",
+     'features': _("To have an icon theme based on FontAwesome."),
+     'required_version': QTAWESOME_REQVER},
+    {'modname': "qtpy",
+     'package_name': "qtpy",
+     'features': _("Abstraction layer for Python Qt bindings so that Spyder can run on multiple Qt bindings and versions."),

```

Comment:
Let's change this description to

    Abstraction layer for Python Qt bindings.